src/sentry/static/sentry/app/components/assigneeSelector.tsx

```diff
@@ -129,58 +119,75 @@ const AssigneeSelectorComponent = createReactClass<Props, State>({
       return true;
     }
     return !valueIsEqual(nextState.assignedTo, this.state.assignedTo, true);
-  },
+  }
+
+  componentWillUnmount() {
+    this.unlisteners.forEach(unlistener => unlistener?.());
+  }
+
+  unlisteners = [
+    GroupStore.listen(itemIds => this.onGroupChange(itemIds), undefined),
+    MemberListStore.listen((users: User[]) => {
+      this.handleMemberListUpdate(users);
+    }, undefined),
+  ];
+
+  handleMemberListUpdate = (members: User[]) => {
+    if (members === this.state.memberList) {
+      return;
+    }
+
+    this.setState({memberList: members});
+  };
 
   memberList(): User[] | undefined {
     return this.props.memberList ? this.props.memberList : this.state.memberList;
-  },
+  }
 
-  assignableTeams(): AssignableTeam[] {
-    if (!this.props.id) {
-      return [];
+  onGroupChange(itemIds: Set<string>) {
+    if (!itemIds.has(this.props.id)) {
+      return;
     }
+    const group = GroupStore.get(this.props.id);
+    this.setState({
+      assignedTo: group && group.assignedTo,
+      suggestedOwners: group && group.owners,
```

Comment:
```suggestion
      suggestedOwners: group?.owners,
```

Could these use optional chaining?